scheduler/scheduler-server/src/main/java/org/ow2/proactive/utils/JettyStarter.java

```diff
@@ -137,8 +132,10 @@ public void deployWebApplications(String rmUrl, String schedulerUrl) {
             addWarsToHandlerList(handlerList, defaultVirtualHost);
             server.setHandler(handlerList);
 
-            startServer(server, ProActiveInet.getInstance().getHostname(), restPort, httpProtocol);
+            String schedulerHost = ProActiveInet.getInstance().getHostname();
+            return startServer(server, schedulerHost, restPort, httpProtocol);
         }
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1168)